src/components/organisms/FullScreenCertificate/_components.fullscreen-certificate.scss

```diff
@@ -126,6 +126,11 @@
   }
 }
 
+.buv-c-fullscreen-certificate__certificate-fixed-width {
```

Comment:
I think it would be best to make that and the other one potentially the same class?

Reply:
They don't have exactly the same properties, which is why I separated them.

Reply:
yeah but does the margin affect in fullscreen?

Reply:
Good point. Updated.